config.json

```diff
@@ -36,26 +36,27 @@
       ]
     },
     {
-      "slug": "bowling",
-      "difficulty": 6,
+      "slug": "bank-account",
+      "difficulty": 2,
       "topics": [
```

Comment:
Why is there churn in this file, regarding things that are unrelated to phone number?